File: julia/Ccxt/test/base/test.inArray.jl
using Test
using Ccxt
function testInArray()

    exchange = get(ccxt, Symbol("Exchange"), nothing)(Dict{Symbol, Any}(
        Symbol("id") => "regirock"
    ));
    array = [1, 2, 3];
    @test functions.ccxtruthy(inArray(exchange, 1, array));
    @test functions.ccxtruthy(inArray(exchange, 2, array));
    @test functions.ccxtruthy(inArray(exchange, 3, array));
    @test inArray(exchange, 4, array) == false;
end

File: ts/src/test/base/test.inArray.ts



import assert from 'assert';
import ccxt from '../../../ccxt.js';

function testInArray () {

    const exchange = new ccxt.Exchange ({
        'id': 'regirock',
    });

    const array = [ 1, 2, 3 ];

    assert (exchange.inArray (1, array) === true);
    assert (exchange.inArray (2, array) === true);
    assert (exchange.inArray (3, array) === true);
    assert (exchange.inArray (4, array) === false);
}

export default testInArray;
